catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplication.java

```diff
@@ -198,6 +198,7 @@ private void registerAuthorizer(CatalogApplicationConfig catalogConfig, Environm
     } else {
       LOG.info("Authorizer config not set, setting noop authorizer");
       authorizer = NoopAuthorizer.class.getConstructor().newInstance();
+      authorizer.init(authorizerConf, jdbi);
```

Comment:
authorizerConf will always be null here.
```suggestion
      authorizer.init(null, jdbi);
```